src/main/java/com/questhelper/helpers/quests/perilousmoon/PerilousMoon.java

```diff
@@ -0,0 +1,702 @@
+		talkToNahta = new NpcStep(this, NpcID.NAHTA, new WorldPoint(1424, 9568, 1),
+			"Talk to Nahta in the magic shop.", earthTalisman, waterTalisman);
+		talkToNahta.addDialogStep("Actually, I've been sent by Attala.");
+		talkToBlacksmith = new NpcStep(this, NpcID.BLACKSMITH_13038, new WorldPoint(1448, 9584, 1),
+			"Talk to the blacksmith near the anvils.", enchantedEarthTalisman, enchantedWaterTalisman);
+		talkToBlacksmith.addDialogSteps("I need your help with these talismans.", "Can you help?");
+		enterNeypotzliAfterBlacksmith = new ObjectStep(this, ObjectID.ENTRANCE_51375, new WorldPoint(1439, 9600, 1),
+			"Enter the Neypotzli entrance in the far north of Cam Torum.");
+		talkToAttalaAfterBlacksmith = new NpcStep(this, NpcID.ATTALA, new WorldPoint(1439, 9638, 1),
+			"Talk to Attala near the Monolith.");
+		useTalismansSidebar = new DetailedQuestStep(this, "Use the talismans to find the correct location." +
+			" Earth tells you if it's east/west, and the water talisman tells you north/south.");
+		useTalismansSidebar.addText("Despite how the world map looks, the antechamber counts as being directly in the middle of the 4 areas, so if it says to go south and east when in there, " +
+			"you should go to the ancient shrine, accessed through the prison or the streambound cavern.");
+		useTalismans = new DetailedQuestStep(this,
+			"Use the talismans to find the correct location. More details in the sidebar.", infusedWaterTalisman, infusedEarthTalisman);
+		useTalismansSidebar.addSubSteps(useTalismans);
+		talkToEyatlalli = new NpcStep(this, NpcID.EYATLALLI, "Talk to Eyatlalli.");
+		talkToJessamineAfterEyatlalli = new NpcStep(this, NpcID.JESSAMINE_12865, new WorldPoint(1440, 9639, 1),
+			"Talk to Jessamine near the Monolith again.");
+//
+		enterStreamboundAgain = new ObjectStep(this, ObjectID.ENTRANCE_51376, new WorldPoint(1458, 9650, 1),
+			"Enter the north-east entrance to the streambound cavern.");
+		collectGrub = new ObjectStep(this, ObjectID.GRUBBY_SAPLING, new WorldPoint(1517, 9690, 0),
+			"Collect a grub from the grubby sapling east of the camp.");
+		getHerbloreSupplies = new ObjectStep(this, ObjectID.SUPPLY_CRATES, new WorldPoint(1511, 9695, 0),
+			"Take herblore supplies from the supply crates in the camp. You only need the pestle and mortar.");
+		getHerbloreSupplies.addDialogStep("Take herblore supplies.");
+		useGrubOnPestle = new DetailedQuestStep(this, "Use a pestle and mortar on the moonlight grub.", pestleAndMortar.highlighted(), moonlightGrub.highlighted());
+		getFishingSupplies = new ObjectStep(this, ObjectID.SUPPLY_CRATES, new WorldPoint(1511, 9695, 0),
+			"Take fishing supplies from the supply crates in the camp. You only need the big fishing net and knife.");
+		getFishingSupplies.addDialogStep("Take fishing supplies.");
+		fishBream = new ObjectStep(this, NullObjectID.NULL_51367, new WorldPoint(1521, 9689, 0),
+			"Fish a bream from the stream east of the camp.", bigFishingNet);
```

Comment:
I think it would be a good idea to require a knife here
```suggestion
			"Fish a bream from the stream east of the camp.", bigFishingNet, knife);
```